tests/e2e/run-e2e.ps1

```
@@ -0,0 +1,373 @@
+        -RedirectStandardError  (Join-Path $LogDir 'server.stderr.txt')
+
+    # Wait for the readiness probe (GET api/vicius/master/schema.json → 200)
+    $readyUrl = 'http://localhost:5200/api/vicius/master/schema.json'
+    $deadline = (Get-Date).AddSeconds(60)
+    $ready = $false
+    Write-Host "  Waiting for server readiness at $readyUrl ..."
+    while ((Get-Date) -lt $deadline) {
+        try {
+            $r = Invoke-WebRequest -Uri $readyUrl -UseBasicParsing -TimeoutSec 2 -ErrorAction Stop
+            if ($r.StatusCode -eq 200) { $ready = $true; break }
+        } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)